server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/actor/Actor.java

```diff
@@ -0,0 +1,92 @@
+package de.tum.in.www1.hephaestus.codereview.actor;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.hephaestus.codereview.comment.Comment;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "actors")
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Actor {
+    /**
+     * Unique identifier for a User entity.
```

Comment:
`@GeneratedValue(strategy = GenerationType.IDENTITY)` means we generate this id ourselves on table-insert.

For users I thought it might make sense to leave out Github's ID since `String login` should be unique anyways.

Would you like me to still add the `github_id`?

Reply:
I think we can use `id` everywhere for now instead of `github_id`. In the future we would then just add a `provider_id` field in addition. 

GitLab uses also an integer for the `id` (Bitbucket as far as I can see too) and others are not interesting for the moment. I think this would be the clearest solution.